assets/app/styles/_core.less

```diff
@@ -404,6 +413,14 @@ code {
   white-space: normal;
 }
 
+pre {
+  &.code {
```

Comment:
nevermind I see you added the .code class so this is a pre tag with code class, i doubt we use that anywhere else.  But that still brings up the question of is the existing pre tag style something we are never going to use / do we need to go back to patternfly with suggested style changes